src/main/java/com/google/sps/data/Event.java

```diff
@@ -0,0 +1,184 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+ 
+package com.google.sps.data;
+ 
+/**
+ * Class that creates an event specific to a POI 
+ */
+public class Event {
+ 
+  // inputs
+  private String name;
+  private String address;
+  private String date;
+  private int startTime;
+  private int endTime;
+  private int travelTime;
+
+  // fields to be saved for parsing
+  private String strStartTime;
+  private String strEndTime;
+ 
+  // class constants
+  private final static int HALFHOUR = 30;
+  private final static int HOUR = 60;
+  private final static int MINUTES_IN_A_DAY = 1440;
+  private final static int MIN_POSSIBLE_TIME = 0;
```

Comment:
Nit: According to the style guide, the order of modifiers should be private static final